### DISCUSSION TITLE
Having an issue importing Dropzone

### DISCUSSION THREAD

**Discussion by kkbco** [react-dropzone/react-dropzone#1366] [Q&A]
React: 17
Webpack: 5
react-dropzone: >10 (haven't tried older versions)

imported as: import Dropzone from 'react-dropzone';

error:
`ERROR in ./node_modules/file-selector/dist/es5/file-selector.js 15:11-20
export '__awaiter' (imported as '__awaiter') was not found in 'tslib' (possible exports: default)
 @ ./node_modules/file-selector/dist/es5/index.js 1:0-44 1:0-44
 @ ./node_modules/react-dropzone/dist/es/index.js 30:0-42 378:61-70
 @ ./client/components/Screen.js 21:0-38 1354:44-52
 @ ./client/service.js 2:0-39 3:15-20
 @ ./dev/frame.js 20:0-40 403:43-50

ERROR in ./node_modules/file-selector/dist/es5/file-selector.js 16:15-26
export '__generator' (imported as '__generator') was not found in 'tslib' (possible exports: default)
 @ ./node_modules/file-selector/dist/es5/index.js 1:0-44 1:0-44
 @ ./node_modules/react-dropzone/dist/es/index.js 30:0-42 378:61-70
 @ ./client/components/Screen.js 21:0-38 1354:44-52
 @ ./client/service.js 2:0-39 3:15-20
 @ ./dev/frame.js 20:0-40 403:43-50`
 
 Could someone help me please? Thank you

**Comment by rolandjitsu** [comment]
@kkbco hmm, I think someone else reported a warning related to tslib imports. I'd need to look into this to see if there's some kind of issue.
